.github/scripts/label_utils.py

```diff
@@ -20,11 +20,12 @@
 
 LABEL_ERR_MSG_TITLE = "This PR needs a label"
 LABEL_ERR_MSG = f"""# {LABEL_ERR_MSG_TITLE}
-    If your changes are user facing and intended to be a part of release notes, please use a label starting with `release notes:`.
+If your changes are user facing and intended to be a part of release notes, please use a label starting with `release notes:`.
 
-    If not, please add the `topic: not user facing` label.
-    For more information, see
-    https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work.
```

Comment:
Note that removing the leading spaces is necessary here, as otherwise GitHub formats the part of the multiline string after any line breaks weirdly as a code block, see [this example](https://github.com/Goldspear/pytorch-dummy/pull/3#issuecomment-1427754611) without this change vs the [fixed one] after removing the spaces (https://github.com/Goldspear/pytorch-dummy/pull/3#issuecomment-1427756769)